simulate/simulate.go

```diff
@@ -501,3 +514,33 @@ func RemoveAncestors(filename string, tree *expandedTree.ETree, outputFilename s
 	outFile = outputFilename
 	fasta.Write(outFile, newFastas)
 }
+
+// seqContainsDels checks whether a deletion is present in slice of bases.
+func seqContainsDels(seq []dna.Base, del dna.Base) bool {
```

Comment:
This works, but if you are just checking for dels, do you need the dna.Base input? You could just have `if base == dna.Gap`